scripts/version.sh

```
@@ -0,0 +1,90 @@
+
+set -e
+
+echo "📦 Version Management Script"
+echo "=============================="
+
+# Read current version
+CURRENT_VERSION=$(grep '^version:' pubspec.yaml | sed 's/version: //' | sed 's/+.*//')
+echo "Current version: $CURRENT_VERSION"
+
+# Parse version components
+MAJOR=$(echo $CURRENT_VERSION | cut -d. -f1)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)